deployment/debian-package-armhf/docker-build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Builds the DEB inside the Docker container
+
+set -o errexit
+set -o xtrace
+
+# Move to source directory
+pushd ${SOURCE_DIR}
+
+# Remove build-dep for dotnet-sdk-2.2, since it's not a package in this image
+sed -i '/dotnet-sdk-2.2,/d' debian/control
```

Comment:
can this trickery be done in `debian/control` like other checks for target arch are done in `debian/rules`?..

Reply:
Not easily, no. It's needed because of that dotnet Docker inage not installing a package for dotnet. I hate to even have to do tgis.

Reply:
Ironically yours and @nvllsvm's comments are the exact opposite of each other - this is what's required to make the other image work, and hence why I'm not a big fan of it. But doing this makes no difference if the package actually *is* installed.